### PR TITLE
Remove unnecessary commonjs plugin

### PR DESCRIPTION
**Pull request by wwilsman**
#22

### CODE REVIEW

File: packages/mocha/tests/it-test.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ describe('BigTest Mocha: it', () => {
 
   it('throws on failure before the timeout', () => {
     expect(tests[1].duration).to.be.within(180, 200);
-    expect(tests[1].err).to.have.property('expected', 200);
+    expect(tests[1].err).to.have.property('expected', '200');
```
Comment (Robdel12) [inline packages/mocha/tests/it-test.js]: [Hm](https://media.giphy.com/media/U8ZoygMEH3eBW/giphy.gif)

**Comment by Robdel12** [comment]
So why is change needed? Is this so we don't need to use commonjs to consume our other packages? Or does this impact anyone who is consuming our packages?

**Comment by wwilsman** [comment]
@Robdel12 [`rollup-plugin-commonjs`](https://github.com/rollup/rollup-plugin-commonjs) bundles your CommonJS dependencies in `node_modules`. Our only dependencies so far are `@bigtest/convergence` in `@bigtest/mocha` which now has ES module support with #20.

If we later need dependencies outside of this repo that do not support ES modules, then that package will require this plugin.

**Comment by Robdel12** [comment]
Okay, so it's just an internal change and those who consume these packages on NPM will still get the `umd` output, right?

**Comment by wwilsman** [comment]
Word

**Pull request merged**